lms/services/blackboard_api/_schemas.py

```diff
@@ -8,46 +8,35 @@
 from lms.validation._base import RequestsResponseSchema
 
 
-class _FileSchema(Schema):
-    """Shared base schema for Blackboard file dicts."""
-
-    class Meta:
-        unknown = EXCLUDE
-
-    id = fields.Str(required=True)
-    display_name = fields.Str(data_key="name", required=True)
-    updated_at = fields.Str(data_key="modified", required=True)
-    mime_type = fields.Str(data_key="mimeType")
+class BlackboardListFilesSchema(RequestsResponseSchema):
+    """Schema for Blackboard API /courses/{courseId}/resources responses."""
 
-    @post_load
-    def post_load(self, data, **_kwargs):  # pylint:disable=no-self-use
-        data["id"] = f"blackboard://content-resource/{data['id']}/"
-        return data
+    class FileSchema(Schema):
+        """Schema for individual Blackboard file dicts."""
 
+        class Meta:
+            unknown = EXCLUDE
```

Comment:
The `EXCLUDE` is still here meaning the schema still removes unknown fields from the Blackboard dicts. In the future I think the service will add a query param to the Blackboard API that filters what fields we want to get back to save bytes, at which point this `EXCLUDE` won't be doing anything and could be removed